### PR TITLE
General availability of support for password protected files for Endpoint DLP (Conditions Devices supports) ?

### PR DESCRIPTION
**Pull request by samgas**
General availability of support for password protected files for Endpoint DLP (Conditions Devices supports) ?  
Cf https://techcommunity.microsoft.com/t5/security-compliance-and-identity/new-capabilities-that-help-proactively-secure-data-with/ba-p/3649211

### CODE REVIEW

File: microsoft-365/compliance/dlp-policy-reference.md [hunk 1]
```diff
@@ -310,7 +310,7 @@ The available context options change depending on which location you choose. If
 ##### Conditions Devices supports
 
 - Content contains
-- (preview) Document or attachment is password protected (.pdf, Office files and Symantec PGP encrypted files are fully supported).This predicate doesn’t detect digital rights managed (DRM) encrypted or permission protected files.
+- Document or attachment is password protected (.pdf, Office files and Symantec PGP encrypted files are fully supported).This predicate doesn’t detect digital rights managed (DRM) encrypted or permission protected files.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-policy-reference.md:313]:
```suggestion
- Document or attachment is password protected (.pdf, Office files, and Symantec PGP encrypted files are fully supported). This predicate doesn’t detect digital rights managed (DRM) encrypted or permission-protected files.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @samgas, kindly commit the minor suggested changes. Many thanks.

**Comment by yogkumgit** [comment]
@samgas Please implement the suggested changes. Thanks

**Comment by chrfox** [comment]
@samgas Pending notification from the PG, this is still in public preview

**Comment by chrfox** [comment]
closing, this is incorrect

**Pull request closed without merging**